Libraries/LibWeb/CSS/Parser/PropertyParsing.cpp

```diff
@@ -2560,12 +2560,12 @@ RefPtr<CSSStyleValue const> Parser::parse_font_value(TokenStream<ComponentValue>
             line_height.release_nonnull(),
 
             // Reset implicitly
-            initial_value, // font-feature-settings
-                           // FIXME: font-kerning,
-            initial_value, // font-language-override
-                           // FIXME: font-optical-sizing,
-                           // FIXME: font-size-adjust,
-            initial_value, // font-variation-settings
+            initial_value,                                   // font-feature-settings
+            property_initial_value(PropertyID::FontKerning), // font-kerning,
```

Comment:
Using `initial_value` here caused a crash in `ComputedProperties::font_kerning()`, which expected a valid FontKerning keyword value. Changing the other properties here to do the same thing didn't make any difference, so I left them as they were.

Reply:
Hmm, `initial` shouldn't still be around by then. But font properties get computed a bit differently from everything else so maybe that's muddling things up.